### PR TITLE
Fix lookup fulltext document not finding files#

### LINKED ISSUE #3874: Look up fulltext documents does not link existing PDFs
It looks like the autolink functionality does not work as expected anymore:

1. I already have a file starting with the correct BibTeX key in my PDF repo
2. It still downloads the file from the web

I expect to pick up the file from the PDF repo instead.

### PR DESCRIPTION
**Pull request by PedroPerozin**
Fixes  #3874 

----

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [x] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

File: src/main/java/org/jabref/gui/externalfiles/FindFullTextAction.java [hunk 1]
```diff
@@ -70,14 +72,20 @@ public void execute() {
...
             protected Map<BibEntry, Optional<URL>> call() {
                 Map<BibEntry, Optional<URL>> downloads = new ConcurrentHashMap<>();
                 int count = 0;
+                AutoSetFileLinksUtil util = new AutoSetFileLinksUtil(basePanel.getBibDatabaseContext(), Globals.prefs.getFilePreferences(), Globals.prefs.getAutoLinkPreferences(), ExternalFileTypes.getInstance());
+                util.linkAssociatedFiles(basePanel.getSelectedEntries(), new NamedCompound(""));
+
                 for (BibEntry entry : basePanel.getSelectedEntries()) {
-                    FulltextFetchers fetchers = new FulltextFetchers(Globals.prefs.getImportFormatPreferences());
-                    downloads.put(entry, fetchers.findFullTextPDF(entry));
+                    List<LinkedFile> l = entry.getFiles();
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/externalfiles/FindFullTextAction.java]: Please no single character or other abbreviation. Makes it harder for others to understand

Comment (PedroPerozin) [reply src/main/java/org/jabref/gui/externalfiles/FindFullTextAction.java]: Ok, sorry. It's my first contribution, i will change.

**Comment by stefan-kolb** [comment]
What is the current status here 😺

**Comment by Siedlerchr** [comment]
Superseeded by #5216

**Pull request closed without merging**